velox/docs/functions/spark/decimal.rst

```diff
@@ -9,6 +9,9 @@ https://cwiki.apache.org/confluence/download/attachments/27362075/Hive_Decimal_P
 
 https://msdn.microsoft.com/en-us/library/ms190476.aspx
 
+Additionally, the computation of decimal division adapts to the allowing precision loss parameter,
```

Comment:
@jinchengchenghh I'm seeing that PR description says "Each of the decimal operation functions is registered as two functions such as add_deny_precision_loss and add." But this doc suggests that only "divide" operation has 2 modes. These two seem to contradict each other. Would you clarify?

Reply:
The total computation result type is divided to 2 parts, this is the first stage, compute the result type, division is different with others when not allowing precision loss, `adjust` or `bounded` the computed result type in the second stage, all operations differ with allow precision loss or not.